src/engine/strat_engine/thinpool/thinpool.rs

```diff
@@ -283,16 +284,20 @@ impl ThinPool {
                     }
                 }
 
-                let usage = &status.usage;
-                if usage.used_meta > usage.total_meta - META_LOWATER {
-                    // TODO: Extend meta device
+                if status.usage.used_meta >
```

Comment:
This is minor, but I liked stowing ```status.usage``` in ```usage```. Reduces line length a little.

Reply:
Ok - I'll put it back.